docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -169,13 +169,15 @@ Some possibly useful slots:
 
 - `appTitle`: allows you to customize the app title section in the layout header.
 
-- `toolbarActions`: allows you to add new items to the toolbar in the header, such as a search bar or button. The default `ThemeSwitcher` component can be imported and used if you wish to do so, as shown in the example below.
+- `toolbarActions`: allows you to add new items to the toolbar in the header, such as a search bar or button. The default `ThemeSwitcher` and `Account` component can be imported and used if you wish to do so, as shown in the example below.
 
-- `sidebarFooter`: allows you to add footer content in the sidebar.
+- `appBar`: allows you to fully replace the layout header. This slot gives you full control over the toolbar's structure while still allowing you to use the built-in components like `AppTitle`, `ToolbarActions`, and `Account`.
 
-{{"demo": "DashboardLayoutSlots.js", "height": 400, "iframe": true}}
+- `sidebarFooter`: allows you to add a footer to the sidebar.
+
+### sidebarFooter
 
-### Examples
```

Comment:
I think you mistakenly removed this header.

Reply:
Alo the example "User account in layout sidebar" will need a slight update as its using the `toolbarAccount` slot that we're deprecating.